runtime/tests/checker/typeargument_test.go

```diff
@@ -178,6 +155,79 @@ func TestCheckTypeArguments(t *testing.T) {
 	})
 }
 
+type checkParameterizedTypeIsInstantiatedTestCase struct {
+	name string
+	code string
+}
+
+func TestCheckParameterizedTypeIsInstantiated(t *testing.T) {
+
+	t.Parallel()
+
+	baseValueActivation := sema.NewVariableActivation(sema.BaseValueActivation)
+	baseValueActivation.DeclareValue(stdlib.InclusiveRangeConstructorFunction)
+	options := ParseAndCheckOptions{
+		Config: &sema.Config{
+			BaseValueActivation: baseValueActivation,
+		},
+	}
+
+	runTestCase := func(t *testing.T, testCase checkParameterizedTypeIsInstantiatedTestCase) {
+		t.Run(testCase.name, func(t *testing.T) {
+
+			t.Parallel()
+
+			_, err := ParseAndCheckWithOptions(t,
+				testCase.code,
+				options,
+			)
+
+			errs := RequireCheckerErrors(t, err, 1)
+
+			assert.IsType(t, &sema.MissingTypeArgumentError{}, errs[0])
+		})
+	}
+
+	testCases := []checkParameterizedTypeIsInstantiatedTestCase{
```

Comment:
Please add more test cases to cover all paths of `checkParameterizedTypeIsInstantiated`, e.g. composite type declarations, enum type declarations, etc.

Reply:
Added more test cases. Can you please take a look and suggest if there is anything else I should add? Thanks!